src/reference/antora/modules/ROOT/pages/amqp/connections.adoc

```diff
@@ -28,6 +28,9 @@ Simple publisher confirmations are supported by all three factories.
 When configuring a `RabbitTemplate` to use a xref:amqp/template.adoc#separate-connection[separate connection], you can now, starting with version 2.3.2, configure the publishing connection factory to be a different type.
 By default, the publishing factory is the same type and any properties set on the main factory are also propagated to the publishing factory.
 
+Starting with version 3.1, the `AbstractConnectionFactory` includes the `connectionCreatingBackOff` property, which supports a backoff policy in the connection module.
+Currently, this support in the behavior of `createChannel` to address exceptions when the channelMax limit is reached.
```

Comment:
The `channelMax` is supposed to be present as a code.
Would you mind mention also that this give a simple retry over `createChannel()` attempts?
Please, add `()` into that `createChannel` since it is a method.

Reply:
in my opinion the backoff policy contains the attempts  and interval, maybe refer the `BackOff `